### PR TITLE
Add reward information to the repository and challenges

### PR DESCRIPTION
**Pull request by celinval**
I also postponed the end date of all challenges, and fixed the date format.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 and MIT licenses.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,24 +7,23 @@
...
 
 The goal is to have a verified [Rust standard library](https://doc.rust-lang.org/std/) and prove that it is safe.
 1. Contributing to the core mechanism of verifying the rust standard library
 2. Creating new techniques to perform scalable verification
 3. Apply techniques to verify previously unverified parts of the standard library.
 
-## [Kani](https://github.com/model-checking/kani)
+For that we are launching a contest that includes a series of challenges that focus on verifying
+memory safety and a subset of undefined behaviors in the Rust standard library.
+Each challenge describes the goal, the success criteria, and whether it has a financial award to awarded upon its
```
Comment (feliperodri) [inline README.md:20]:
```suggestion
Each challenge describes the goal, the success criteria, and whether it has a financial award to be awarded upon its
```

File: README.md [hunk 2]
```diff
@@ -7,24 +7,23 @@
...
+successful completion.
 
-The Kani Rust Verifier is a bit-precise model checker for Rust.
-Kani verifies:
-* Memory safety (e.g., null pointer dereferences)
-* User-specified assertions (i.e `assert!(...)`)
-* The absence of panics (eg., `unwrap()` on `None` values)
-* The absence of some types of unexpected behavior (e.g., arithmetic overflows).
+See [our book](https://model-checking.github.io/verify-rust-std/intro.html) more details on the challenge rules
```
Comment (feliperodri) [inline README.md:23]:
```suggestion
See [our book](https://model-checking.github.io/verify-rust-std/intro.html) for more details on the challenge rules
```

File: doc/src/challenges/0005-linked-list.md [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 - **Status:** Open
 - **Tracking Issue:** [#29](https://github.com/model-checking/verify-rust-std/issues/29)
-- **Start date:** *24/07/01*
-- **End date:** *24/12/10*
+- **Start date:** *2024/07/01*
+- **End date:** *2025/04/10*
+- **Reward:** *5,000 USD*
```
Comment (feliperodri) [inline doc/src/challenges/0005-linked-list.md:7]:
```suggestion
- **Reward:** *N/A*
```

File: doc/src/challenges/0007-atomic-types.md [hunk 4]
```diff
@@ -2,8 +2,9 @@
 
 - **Status:** Open
 - **Tracking Issue:** [#83](https://github.com/model-checking/verify-rust-std/issues/83)
-- **Start date:** *2024-10-30*
-- **End date:** *2024-12-10*
+- **Start date:** *2024/10/30*
+- **End date:** *2025/04/10*
+- **Reward:** *10,000 USD*
```
Comment (feliperodri) [inline doc/src/challenges/0007-atomic-types.md:7]:
```suggestion
- **Reward:** *N/A*
```

File: doc/src/intro.md [hunk 5]
```diff
@@ -6,12 +6,22 @@ library](https://doc.rust-lang.org/std/). The goal of this is
 to provide automated verification that can be used to verify that a
 given Rust standard library implementation is safe.
 
+Verifying the Rust libraries is difficult because:
+1. Lack of a specification,
+2. Lack of an existing verification mechanism in the Rust ecosystem,
+3. Lhe large size of the verification problem,
```
Comment (feliperodri) [inline doc/src/intro.md:12]:
```suggestion
3. The large size of the verification problem,
```

File: doc/src/challenges/0008-smallsort.md [hunk 6]
```diff
@@ -2,8 +2,9 @@
 
 - **Status:** Open
 - **Tracking Issue:** [#56](https://github.com/model-checking/verify-rust-std/issues/56)
-- **Start date:** *2024-08-17*
-- **End date:** *2024-12-10*
+- **Start date:** *2024/08/17*
+- **End date:** *2025/04/10*
+- **Reward:** *10,000 USD*
```
Comment (feliperodri) [inline doc/src/challenges/0008-smallsort.md:7]:
```suggestion
- **Reward:** *N/A*
```

**Review by feliperodri — APPROVED** [review]

**Review by nerryc — APPROVED** [review]

**Review by rahulku — APPROVED** [review]

**Review by celinval — COMMENTED** [review]
Trigger workflow

**Pull request merged**